src/main/kotlin/org/stellar/walletsdk/utils/buildTransaction.kt

```diff
@@ -16,7 +17,7 @@ fun buildTransaction(
 
   try {
     sourceAccount = server.accounts().account(sourceAddress)
-  } catch (e: Error) {
+  } catch (e: Exception) {
     throw Error("Source account was not found")
```

Comment:
This may not be accurate because the `Exception` is an `IOException` which can happen while making HTTP connections to the Horizon server.